.github/workflows/docker.yml

```diff
@@ -2,12 +2,19 @@ name: Publish Docker Image
 
 on:
   push:
-    branches:
-      - main
```

Comment:
though I would say if this job produces a docker image, you probably specifically want it on pushes to `main` as well

Reply:
done